bindings/cpu_profiler.cc

```diff
@@ -100,13 +100,13 @@ v8::Local<v8::Value> CreateFrameGraph(const CpuProfileNode* node) {
 
 #if PROFILER_FORMAT == FORMAT_SAMPLED || FORMAT_BENCHMARK == 1
 v8::Local<v8::Object> CreateFrameNode(
-  v8::Local<v8::String> name, v8::Local<v8::String> scriptName, v8::Local<v8::Integer> line,
+  v8::Local<v8::String> name, v8::Local<v8::String> absPath, v8::Local<v8::Integer> line,
   v8::Local<v8::Integer> column, std::vector<v8::CpuProfileDeoptInfo> deoptInfos) {
 
   v8::Local<v8::Object> js_node = Nan::New<v8::Object>();
 
   Nan::Set(js_node, Nan::New<v8::String>("name").ToLocalChecked(), name);
-  Nan::Set(js_node, Nan::New<v8::String>("file").ToLocalChecked(), scriptName);
+  Nan::Set(js_node, Nan::New<v8::String>("absPath").ToLocalChecked(), absPath);
   Nan::Set(js_node, Nan::New<v8::String>("line").ToLocalChecked(), line);
   Nan::Set(js_node, Nan::New<v8::String>("column").ToLocalChecked(), column);
```

Comment:
The new names for this should be
- `name` -> `function`
- `file` -> `filename`
- `absPath` -> `abs_path` (the camel case variant is actually wrong and will be ignored)
- `line` -> `lineno`
- `column` -> `colno`

Reply:
See https://github.com/getsentry/relay/blob/f0d3edc54b63f4b75f6b8c50a488369d8b2275b0/relay-profiling/src/sample.rs#L16-L33 for the full list. It should match the field names. Try not to use the aliases specified since we're trying to move away from that.

Reply:
Ah, did not know they were aliases, thanks for the pointer. I fixed them up + added in_app bool